catalyst-gateway/bin/src/service/api/cardano/rbac/chain_root_get.rs

```diff
@@ -40,12 +41,14 @@ pub(crate) type AllResponses = WithErrorResponses<Responses>;
 pub(crate) async fn endpoint(stake_address: Cip19StakeAddress) -> AllResponses {
     let Some(session) = CassandraSession::get(true) else {
         error!("Failed to acquire db session");
```

Comment:
Do we need to log an error here? since it is already log in `service_unavailable`
for `internal_error` too

Reply:
How do we know where the error came from?

Reply:
Its fine for now.